cmd/main.go

```diff
@@ -1,4 +1,40 @@
 package main
```

Comment:
Could we move this into a `./cmd/metro2` directory? That way the installed binary (from `go install`) is automatically named `metro2` rather than `cmd`.

Reply:
yep, And I am using this branch as test branch